spec/lib/generators/migration_specs_generator_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'spec_helper'
+require 'generators/migration/migration_specs_generator'
+
+describe ManageIQ::Schema::MigrationSpecsGenerator do
+  include Rails::Generators::TestCase
+  
+  before do 
+    tests MigrationSpecsGenerator
+    destination Rails.root.join('tmp/generators')
+    prepare_destination
+  end
+
+  it "runs the generator without errors" do
+    expect {
+      run_generator
+    }.not_to raise_error
```

Comment:
Not entirely sure these specs even would run (if I didn't have the old model names here).

Will see if they do, but if not, I will probably delete this file.

Reply:
Okay, got this working now.  Resolving.